packages/flutter/lib/src/animation/curves.dart

```diff
@@ -522,16 +522,136 @@ class Curves {
   /// {@animation 464 192 https://flutter.github.io/assets-for-api-docs/assets/animation/curve_ease_in.mp4}
   static const Cubic easeIn = Cubic(0.42, 0.0, 1.0, 1.0);
 
+  /// A cubic animation curve that starts slowly and ends quickly.
+  ///
+  /// Derived from Robert Penner’s easing functions.
```

Comment:
Please add a dartdoc tag for each of these as well - you can see the file names in the linked PR in assets-for-api-docs.  e.g., this one should be:

```
{@animation 464 192 https://flutter.github.io/assets-for-api-docs/assets/animation/curve_ease_in_sine.mp4}
```

Reply:
(This will add an animation of the curve type to our docs, but I still think we should have some kind of prose around what differentiates these curves)